Dockerfile

```diff
@@ -35,4 +35,4 @@ COPY --from=builder /usr/src/app/ursa /usr/local/bin
 # run ursa node
 ENV RUST_LOG=info
 
-ENTRYPOINT ["ursa"]
```

Comment:
What's the change here?

Reply:
editor inserting newline at end of file i think